### PR TITLE
fix(HeightAnimation): enhance calculation of height

### PR DESCRIPTION
**Pull request by tujoworker**
By using `position: absolute` on the first pain styles, we got a wrong height calculation.

### CODE REVIEW

**Review by joakbjerk — DISMISSED** [review]

**Review by joakbjerk — COMMENTED** [review]
Wait I missclicked the approve button 😅 [the keep in DOM ](https://eufemia-git-fix-height-animation-calculation-eufemia.vercel.app/uilib/components/height-animation/#keep-in-dom) does not seem to work anymore

**Comment by tujoworker** [comment]
Nice 👍 can you have a new look?

File: packages/dnb-eufemia/src/components/height-animation/HeightAnimationInstance.ts [hunk 1]
```diff
@@ -181,13 +179,17 @@ export default class HeightAnimation {
...
     this.elem.parentNode?.insertBefore(clonedElem, this.elem.nextSibling)
 
     for (const key in this.firstPaintStyle) {
       clonedElem.style[key] = this.firstPaintStyle[key]
     }
-    clonedElem.style.height = 'auto'
+    clonedElem.style.width = width ? `${String(width)}px` : 'auto' // set width because of the "position: absolute"
+    clonedElem.style.position = 'absolute' // not a part of the "firstPaintStyle"
+
+    // console.log('clonedElem.style.position', clonedElem.style.position)
```
Comment (joakbjerk) [inline packages/dnb-eufemia/src/components/height-animation/HeightAnimationInstance.ts:192]:
```suggestion
```

**Review by joakbjerk — APPROVED** [review]
Looks great now 🙏

**Pull request merged**